kubernetes.go

```diff
@@ -33,14 +33,20 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
 	endpoint "sigs.k8s.io/external-dns/endpoint"
+
 	// "k8s.io/client-go/tools/clientcmd"
```

Comment:
orphan?

Reply:
nope, I use it from time to time when running controller locally for debug or feature implementation